com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -4566,6 +4566,16 @@ private static void OnExport ()
         public static string ExportObject (
             string filePath, 
             UnityEngine.Object singleObject,
+            ExportModelOptions exportOptions = null
+        )
+        {
+            return ExportObjects(filePath, new Object[] { singleObject }, exportOptions.ConvertToModelSettingsSerialize(), exportData: null);
+        }
+
+        [SecurityPermission(SecurityAction.LinkDemand)]
```

Comment:
what does this do?

Reply:
I don't remember but it was to fix a FDG analysis warning/error

Reply:
It looks like it's obsolete in modern C# ; if we don't need it, I'd remove it.

Reply:
from everywhere or just the public ones?

Reply:
I removed all of them